main.py

```diff
@@ -719,7 +719,8 @@ def ui_access_wrapper(self, *args, **kwargs):
     get_redirect_route(
         r'/explorationdataextractionhandler', admin.DataExtractionQueryHandler),
     get_redirect_route(
-        r'/sendDummyMailToAdminHandler', admin.SendDummyMailToAdminHandler),
+        r'/senddummymailtoadmin', admin.SendDummyMailToAdminHandler),
```

Comment:
Should have "handler" at the end, ditto below.

Reply:
Done.